meson.build

```diff
@@ -0,0 +1,175 @@
+project(
+    'cloud-init',
+    version: run_command('./tools/read-version', check: true).stdout().strip(),
+    meson_version: '>=0.63.0',  # rockylinux/9
+    default_options: [
+      # The default can yield broken results.
+      'python.install_env=auto'
+    ]
+)
+
+
+system = host_machine.system()
+libdir = get_option('libdir')
+pkgdatadir = get_option('datadir')
+sysconfdir = get_option('sysconfdir')
+INIT_SYSTEM = get_option('init_system')
+VALID_INIT_SYSTEMS = ['systemd']
+if not VALID_INIT_SYSTEMS.contains(INIT_SYSTEM)
+  error(
+    'Invalid init_system supplied: "@0@". Must be one of: @1@'.format(
+      INIT_SYSTEM, VALID_INIT_SYSTEMS)
+  )
+endif
```

Comment:
The `systemd` option is effectively a boolean switch, so there is nothing to verify.